### DISCUSSION TITLE
Export to LISP data or some other better format.

### DISCUSSION THREAD

**Discussion by gnusupport** [andmarti1424/sc-im#523] [Ideas]
I would like export to be somewhat better like LISP data. But I do understand that project is small and that would really be too much. 

Something like:

(:cell "A01" :type 'left-string "Some string")

**Comment by andmarti1424** [comment]
Anyone know an ods library for C? Thanks

**Reply by akbarnes** [reply]
I don't, but the impression I got was that there are not too many corner cases in S-expressions so it's not hard to write an S-expression generator from [scratch](https://rosettacode.org/wiki/S-expressions)
